pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=61.0.0", "wheel"]
+requires = ["setuptools>=61.0.0"]
```

Comment:
As discussed in the other PR, I would be really surprised if we got issues, considering we switched to `build`